src/NClient.Providers/NClient.Providers.Serialization.SystemXml/NClient.Providers.Serialization.SystemXml.csproj

```diff
@@ -48,7 +48,7 @@
 
     <ItemGroup>
         <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
-            <_Parameter1>NClient.Providers.Serialization.Xml.System.Tests</_Parameter1>
+            <_Parameter1>NClient.Providers.Serialization.SystemXml.Tests</_Parameter1>
```

Comment:
This changes should be in this PR? I don't see any valuable reason to change smth except JSONProviders, do I?

Reply:
I fixed the error, not renamed it. Are you suggesting to create a separate PR for this?)

Reply:
i don't care, to be honest =) Maybe this would be more semantically correct to create separate PR.